README.md

```diff
@@ -267,7 +267,228 @@ export default {
+
+In order to make this task easier, `FormVueLatte` provides a core plugin called `@formvuelatte/plugin-lookup`.
+
+The [repo for the plugin can be found here](https://github.com/vuelidate/formvuelatte-plugin-lookup).
+
+#### Installation
+
+To install the plugin, simply add it to your `package.json` via terminal.
+
+```bash
+yarn add @formvuelatte/plugin-lookup
+
+// OR
+
+npm i @formvuelatte/plugin-lookup
+```
+
+#### Usage
+
+To use the plugin, first import both the plugin itself, and the `SchemaFormFactory` to your application.
+
+```js
+import { SchemaFormFactory } from 'formvuelatte'
+import LookupPlugin from '@formvuelatte/plugin-lookup'
+```
+
+Now that we have both imported, we can create our plugin-enabled `SchemaForm` component by using the `SchemaFormFactory`
+
+```js
+const SchemaFormWithPlugin = SchemaFormFactory([
+  LookupPlugin({
+    // plugin configuration here
+  })
+])
+```
+
+Now that we have created our new component, we can pass it to our instance's `components` object, and use it as we normally would in our template.
+
+```js
+export default {
+  name: 'App',
+  components: {
+    SchemaFormWithPlugin
+  },
+  setup () {
+    [...]
+  }
+}
+```
+
+```html
+<template>
+  <div id="app">
+    <SchemaFormWithPlugin
+      :schema="mySchema"
+      v-model="myData"
+    />
+  </div>
+</template>
+```
+
+**Important: ** Remember that `SchemaFormFactory` returns an extended version of `SchemaForm`, so all the props required by `SchemaForm` like `schema` and `modelValue`/`v-model` are still required.
+
+#### Configuration
+
+`LookupPlugin` takes one parameter, an object, as it's source of configuration.
+Let's look at the properties that we can use in this object.
+
+**componentProp**
+
+`SchemaForm` schemas expect each component inside of them to be defined with a `component` property, like in the following example.
+
+```json
+{
+  "firstName": {
+    "component": "FormText",
+    "label": "First name"
+  }
+}
+```
+
+In some cases the schema might define your `component` property with something else, like `type` like in the following example:
+
+```json
+{
+  "firstName": {
+    "type": "FormText",
+    "label": "First name"
+  }
+}
+```
+
+If this is the case, you can pass into the configuration the `componentProp` property with the name of what YOUR schema uses to define the component for each node.
+
+```js
+LookupPlugin({
+  componentProp: 'type'
+})
+```
+
+The plugin will handle parsing the schema from `type` into `component` for you now.
+
+**mapComponents**
+
+If your schema does not provide component names as your Vue application needs them, `mapComponents` is another property of the configuration object that can allow you to rename or remap these values with ease.
+
+Consider the following example schema.
+
+```json
+{
+  "firstName": {
+    "component": "string",
```

Comment:
I think this is a typo. Shouldn’t it be `"type": "string"` considering the configuration below?

Reply:
Each example is individual and does not build on itself on purpose

Reply:
I see. Well, I was confused here in that case. I think it would still be better to say `type`, since it just makes more sense. `type: 'string'` or `type: 'array'` sounds more like real-life.
I mean, I don’t think I would have had a schema where I had a field `component: 'array'`.